ui/raidboss/data/05-shb/raid/e1s.js

```diff
@@ -578,7 +578,7 @@
         'Paradise Regained': '复乐园',
         'Pure Beam': '净土射线',
         'Pure Light': '净土之光',
-        'Spear [oO]f Paradise': '乐园之枪',
```

Comment:
oof

Reply:
there is nothing here you have to look at <.<